registry/internal/tools/finder/finder.go

```diff
@@ -29,6 +29,13 @@ type Tag struct {
 	Commit string
 }
 
+type SemVer struct {
```

Comment:
Noted below, i think we can make use of https://github.com/shoenig/semantic/blob/master/tag.go#L68